packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -33,6 +33,8 @@ const eightOfTwelveColumnsMaxScaleable = `66.67%`;
 // (8 / 12) * 100 = 66.6666666667 = 66.67%
 
 const StoryPromoWrapper = styled.div`
+  position: relative;
```

Comment:
What's the reason this is needed for?

Reply:
Ahhh is it part of faux block link?

Reply:
That is used to make the `faux-block` pattern work. If you don't apply it the clickable area is everything and not just the `StoryPromoWrapper`.